site/styles.css

```diff
@@ -193,17 +272,70 @@ input[type="submit"]:hover {
 
 .button-container {
     display: flex;
+    justify-content: center;
 }
 
 .button-container .button {
+    display: inline-block;
     margin-right: 16px;
 }
 
 .button-container .button:last-child {
     margin-right: 0;
 }
 
+.back-to-top {
+    opacity: 1;
+    pointer-events: all;
+    position: fixed;
+    bottom: 6rem;
```

Comment:
This feels a little high up the page when I'm using it, could we knock it down a little?